File: Exemplos M10/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/SAT/Commands/ExtrairLogs.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.SAT.Commands

import com.elgin.intent_digital_hub.IntentDigitalHubService.SAT.SatCommand

class ExtrairLogs(private val numSessao: Int, private val codAtivacao: String) :
    SatCommand("ExtrairLogs") {
    override fun functionParameters(): String {
        return "\"numSessao\"" + ":" + numSessao + "," +
                "\"codAtivacao\"" + ":" + "\"" + codAtivacao + "\""
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/SAT/extrair_logs.dart
import 'package:intent_digital_hub/IntentDigitalHubService/SAT/sat_command.dart';

class ExtrairLogs extends SatCommand {
  final int _numSessao;
  final String _codAtivacao;

  ExtrairLogs(this._numSessao, this._codAtivacao) : super('ExtrairLogs');

  @override
  functionParameters() {
    return '"numSessao"'
        ':'
        '$_numSessao'
        ','
        '"codAtivacao"'
        ':'
        '"$_codAtivacao"';
  }
}
